runelite-client/src/main/java/net/runelite/client/plugins/combatlevel/CombatLevelOverlay.java

```diff
@@ -47,18 +50,24 @@
 
 public class CombatLevelOverlay extends Overlay
 {
+	private static final int OVERLAY_MIN_WIDTH = 140;
+
 	private final Client client;
-	private final PanelComponent panelComponent;
 	private final CombatLevelConfig config;
 
+	private final PanelComponent combatLevelTooltip;
+
 	@Inject
 	CombatLevelOverlay(Client client, CombatLevelConfig config)
 	{
+		setPosition(OverlayPosition.DYNAMIC);
+		setPriority(OverlayPriority.HIGH);
+		setLayer(OverlayLayer.ALWAYS_ON_TOP);
 		this.client = client;
 		this.config = config;
-		this.panelComponent = new PanelComponent();
-		panelComponent.setTitle("Next Combat Level");
-		panelComponent.setTitleColor(Color.WHITE);
+		this.combatLevelTooltip = new PanelComponent();
```

Comment:
Why not use the `TooltipManager` here, instead of building and managing your own PanelComponent?

Reply:
I saw that in use in the mouse highlight plugin but I thought it was geared more towards "simple" tooltips. I'll have another look in the morning 👍

Reply:
I believe the item stats plugin has some multi-line tooltips if you want example code